plugins/basic/python/tk_3dsmaxplus_basic/plugin_bootstrap.py

```diff
@@ -18,10 +18,9 @@
 from . import __name__ as PLUGIN_PACKAGE_NAME
 
 try:
-    from PySide2 import QtCore
+    from PySide6 import QtCore
 except ImportError:
-    # Max 2017 was PySide 1, so fallback on that
-    from PySide import QtCore
+    from PySide2 import QtCore
```

Comment:
I am not happy about this strategy. I think I raised the same comment for VRED. 
I would prefer that we import the proper version of PySide according to the Max version instead.

Reply:
But.... this was already working so this is a logical change.

Reply:
Yes, in this file it seems that the `import sgtk` line was not added at the beginning of the file for a reason (dynamic patching of the python path or something like that).